### ISSUE TITLE
[Game Support]: I get an error when installing ipa.(FGO)

### ISSUE THREAD

**Issue by Paimonm** [PlayCover/PlayCover#712]
### App Store link

https://apps.apple.com/kr/app/페이트-그랜드-오더/id1241893973

### Current issues

- [X] Game fails to install
- [ ] Game fails to open
- [ ] Game crashes randomly during gameplay
- [ ] Game does not work with keymapping
- [ ] Game experiences graphical issues or broken UI
- [ ] Game fails to connect to servers or complete download

### More details

I get an error when installing ipa.

### Crash log

```shell
I am getting an error at step "Creating app container".

Message(ko/en):
ko: 이전 항목이 없거나 해당 문자를 포함하는 폴더가 존재하지 않기 때문에 ‘fgok.app’을(를) ‘페이트’(으)로 이동할 수 없습니다.
en: Cannot move 'fgok.app' to 'Fate' because there is no previous entry or the folder containing that character does not exist.
```

### Issue Language

- [X] Yes my issue is written in English

**Comment by TheMoonThatRises** [comment]
This issue was discussed in #399 and fixed in #685. You can use the nightly build to install FGO.

**Comment by Paimonm** [comment]
@TheMoonThatRises The problem has been resolved. thank you(v2.0.3)
